app/src/main/java/helium314/keyboard/latin/suggestions/SuggestionStripView.java

```diff
@@ -417,10 +420,10 @@ private boolean onLongClickSuggestion(final TextView wordView) {
                 showIcon = false;
         }
         if (showIcon) {
-            final Drawable icon = mBinIcon;
+            final Drawable icon = mSuggestedWords.isClipboardSuggestion() ? mCloseIcon : mBinIcon;
             Settings.getInstance().getCurrent().mColors.setColor(icon, ColorType.REMOVE_SUGGESTION_ICON);
-            int w = icon.getIntrinsicWidth();
-            int h = icon.getIntrinsicWidth();
+            int w = wordView.getWidth();
+            int h = wordView.getHeight();
```

Comment:
Does this still allow selecting a word once the bin icon is showing?

Reply:
Selecting a word once the bin icon is shown will cause the suggestion to be deleted.
I made this change because it seemed too easy to accidentally pick the (clipboard) suggestion when trying to delete it.

Reply:
In [d4083a3](https://github.com/Helium314/HeliBoard/pull/647/commits/d4083a386d87c8cc98f8a6f8b6448af05451a75c) I reverted this change as I figured out what was causing the suggestions to be accidentally picked.
OnLongClick() would not return true if the bin icon had not been touched, so the click event was being consumed by OnClick()